src/app/pages/MostWatchedPage/MostWatchedPage.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { node } from 'prop-types';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MAX,
+  GEL_GROUP_5_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_TRPL,
+  GEL_SPACING_QUAD,
+} from '@bbc/gel-foundations/spacings';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Grid, { GelPageGrid } from '#app/components/Grid';
+import IndexMain from '#app/components/PageLayout/IndexMain';
+import IndexPageContainer from '#app/components/PageLayout/IndexPageContainer';
+import IndexHeading from '#containers/IndexHeading';
+
+const MarginWrapper = styled.div`
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    margin-top: ${GEL_SPACING_TRPL};
+  }
+
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
+    margin-top: ${GEL_SPACING_QUAD};
+  }
+
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    margin-top: ${GEL_SPACING_QUAD};
+  }
+`;
+
+const MostWatchedPage = () => {
+  const {
+    dir,
+    mostWatched: { header },
+  } = useContext(ServiceContext);
+
+  const MostWatchedWrapper = ({ children }) => (
+    <>
+      <IndexHeading id="content">{header}</IndexHeading>
+      <MarginWrapper>
+        <GelPageGrid
```

Comment:
We have this in live radio and OD radio pages too, don't we? What do we think about doing something like this?

https://github.com/bbc/simorgh/blob/media-team-collab-week/src/app/containers/PageHandlers/withStickyPlayer/Container.jsx